app/views/layouts/_social_icons.html.erb

```diff
@@ -1,20 +1,46 @@
-<a href="/auth/google_oauth2?origin=<%= params[:return_to] || root_url %>" id="connect-google">
-	<span class="btn btn-default" style="margin-right:2px;background-color: #d34836;">
-		<i class="fa fa-google fa-fw" style="font-size:20px;color:white;"></i>
-	</span>
-</a>
-<a href="/auth/github?origin=<%= params[:return_to] || root_url %>" id="connect-github">
-	<span class="btn btn-default" style="margin-left:2px;background-color: #333;">
-		<i class="fa fa-github fa-fw" style="font-size:20px;color:white;"></i>
-	</span>
-</a>
-<a href="/auth/twitter?origin=<%= params[:return_to] || root_url %>" id="connect-twitter">
-	<span class="btn btn-default" style="margin-left:2px;background-color: #1da1f2;">
-		<i class="fa fa-twitter fa-fw" style="font-size:20px;color:white;margin-left:2px;"></i>
-	</span>
-</a>
-<a href="/auth/facebook?origin=<%= params[:return_to] || root_url %>" id="connect-facebook">
-	<span class="btn btn-default" style="margin-left:2px;background-color: #3b5998">
-		<i class="fa fa-facebook fa-fw" style="font-size:20px;color:white;"></i>
-	</span>
-</a>
+<div class="container">
+	
+	<div class="row text-center">
+		<a href="/auth/google_oauth2?origin=<%= params[:return_to] || root_url %>" id="connect-google">
+			<span class="btn btn-default" style="margin-right:2px;background-color: #d34836;">
+				<i class="fa fa-google fa-fw" style="font-size:20px;color:white;"></i>
+			</span>
+		</a>
+		<a href="/auth/github?origin=<%= params[:return_to] || root_url %>" id="connect-github">
+			<span class="btn btn-default" style="margin-left:2px;background-color: #333;">
+				<i class="fa fa-github fa-fw" style="font-size:20px;color:white;"></i>
+			</span>
+		</a>
+		<a href="/auth/twitter?origin=<%= params[:return_to] || root_url %>" id="connect-twitter">
+			<span class="btn btn-default" style="margin-left:2px;background-color: #1da1f2;">
+				<i class="fa fa-twitter fa-fw" style="font-size:20px;color:white;margin-left:2px;"></i>
+			</span>
+		</a>
+		<a href="/auth/facebook?origin=<%= params[:return_to] || root_url %>" id="connect-facebook">
+			<span class="btn btn-default" style="margin-left:2px;background-color: #3b5998">
+				<i class="fa fa-facebook fa-fw" style="font-size:20px;color:white;"></i>
+			</span>
+		</a>
+	</div>
+	
+	<div class='hr-or'></div>
```

Comment:
Actually @JonathanXu1 has shifted ``OR`` in the _social_icons partial.
So, there are some changes for the sign up form.
We can merge this, I think. Also, the screenshot depicts well the changes does not effect sign up form.